### ISSUE TITLE
Link `staging` branch to deploy on isotc211.geolexica.org `staging` branch

### ISSUE THREAD

**Issue by ronaldtse** [geolexica/isotc211-glossary#26]
When `staging` branch is updated in the current repo, we should trigger an update of the https://github.com/geolexica/isotc211.geolexica.org/ `staging` branch.

**Comment by ronaldtse** [comment]
Thank you @CAMOBAP !

**Comment by CAMOBAP** [comment]
@ronaldtse @strogonoff The issue fixed, but I would like update isotc204.geolexica.org repo also and merge:
- site-deploy-breviter.yml
- site-deploy.yml 

Workflows

**Comment by ronaldtse** [comment]
@CAMOBAP yes please, thanks! Ideally we want all the workflows to be shared across the site and concept repositories.

**Comment by CAMOBAP** [comment]
Simplified for both:
 - https://github.com/geolexica/isotc204.geolexica.org/pull/14
 - https://github.com/geolexica/isotc211.geolexica.org/pull/193
